project/Version.scala

```diff
@@ -1,14 +1,14 @@
 object Version {
   val awscala = "0.8.1"
-  val vectorpipe = "2.0.0-SNAPSHOT"
+  val vectorpipe = "2.1.0-SNAPSHOT"
```

Comment:
I believe this can remain 2.0.0-SNAPSHOT as we haven't yet released an official 2.0 yet.